launcher/ui/pages/global/APIPage.cpp

```diff
@@ -183,6 +185,7 @@ void APIPage::applySettings()
     QString flameKey = ui->flameKey->text();
     s->set("FlameKeyOverride", flameKey);
     s->set("UserAgentOverride", ui->userAgentLineEdit->text());
+    s->set("DisableChecksumVerification", ui->disableChecksumVerificationCheckBox->isChecked());
```

Comment:
Before setting, wouldn't it be *much* better to prompt the user with a warning
not to do this? This is a *really* dangerous thing.